packages/azure-http-specs/specs/azure/resource-manager/resources/extension.tsp

```diff
@@ -0,0 +1,554 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.Resources;
+
+model ExtensionsResource is ExtensionResource<ExtensionsResourceProperties> {
+  ...ResourceNameParameter<ExtensionsResource>;
+}
+
+/** ExtensionsResource properties */
+model ExtensionsResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+/** The interface of extensions resources,
+ * it contains 4 kinds of URLs (resource, resource group, subscription and tenant)
```

Comment:
nit, 4 kinds of scopes

Reply:
Fixed it by the [commit#9ef6412](https://github.com/Azure/typespec-azure/pull/1924/commits/9ef64123b6b361f5a9af867cbb3df8be12ffa92a)